lib/cb/models/implementations/saved_search.rb

```diff
@@ -36,18 +36,19 @@ def initialize(args = {})
         @search_parameters          = SearchParameters.new(args['SavedSearchParameters'] || {})
       end
 
-      def create_to_xml
-        <<-eos
-          <Request>
-            <HostSite>#{host_site}</HostSite>
-            <Cobrand>#{cobrand}</Cobrand>
-            <SearchName>#{search_name}</SearchName>
-            #{search_parameters.to_xml}
-            <IsDailyEmail>#{is_daily_email.to_s.upcase}</IsDailyEmail>
-            <ExternalUserID>#{external_user_id}</ExternalUserID>
-            <DeveloperKey>#{Cb.configuration.dev_key}</DeveloperKey>
-          </Request>
-        eos
+      def create_to_json
+        hash = {
+          'SiteID' => site_id,
+          'Cobrand' => cobrand,
+          'EmailDeliveryDay' => email_delivery_day,
+          'IsDailyEmail' => is_daily_email.to_s.upcase,
+          'userOAuthToken' => user_oauth_token,
+          'HostSite' => host_site,
+          'SearchName' => search_name,
+          'SavedSearchParameters' => search_parameters.to_hash
+        }
+        hash['EmailDeliveryDay'] = email_delivery_day unless is_daily_email
```

Comment:
Gotcha. The way it's written, both values will always be set. First they're both set in the `hash = { ... }` block, then EmailDeliveryDay is conditionally set to the same thing again depending on the nil-ness of IsDailyEmail. Neither property is removed from the hash. If they both always get set, and that works ok, can line 50 be removed since it is conditionally doing the same thing that happens on line 43?
```ruby
yay = 'yo'
it = { boo: 'boo', yay: yay }
it[:yay] = yay unless true
# => {:boo=>"boo", :yay=>"yo"}